manifests/pulp.pp

```diff
@@ -129,6 +142,53 @@
     }
   }
 
+  if $expose_mongodb {
+    include certs::mongodb
+    include certs::mongodb_client
+
+    Class['certs::mongodb_client'] ~> Class['pulp', 'apache::service']
+
+    $mongodb_seeds_real = "${facts['networking']['fqdn']}:27017"
```

Comment:
```suggestion
    $mongodb_seeds_real = "${facts['networking']['fqdn']}:${mongodb::server::port}"
```

Reply:
They are 99.9% of the time going to be the same thing because this is a tightly controlled scenario. But I shall may the change.

Reply:
The use of `mongodb::server::port` is incorrect here since port is undef -- https://github.com/voxpupuli/puppet-mongodb/blob/master/manifests/server.pp#L27

Reply:
> They are 99.9% of the time going to be the same thing because this is a tightly controlled scenario. But I shall may the change.

The important part is showing that these are linked. That's what variables are for. It gives understanding to the reader of the code.

> The use of `mongodb::server::port` is incorrect here since port is undef -- https://github.com/voxpupuli/puppet-mongodb/blob/master/manifests/server.pp#L27

Ah right. Fair enough.